deploy/olm-catalog/community-kubevirt-hyperconverged/1.9.0/manifests/kubevirt-hyperconverged-operator.v1.9.0.clusterserviceversion.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     certified: "false"
     console.openshift.io/disable-operand-delete: "true"
     containerImage: quay.io/kubevirt/hyperconverged-cluster-operator:1.9.0-unstable
-    createdAt: "2022-12-07 12:56:54"
+    createdAt: "2022-12-15 11:09:32"
```

Comment:
aint this field can be totally removed from the CSV ? 
(not related to this PR)

Reply:
no, it's a mandatory field in the CSV